tests/test-match.sh

```diff
@@ -555,3 +555,32 @@ test_that_matchtimeout_screenshot_doesnt_include_visualisation() {
     stbt match screenshot.png "$testdir"/black-full-frame.png \
         match_method=ccoeff-normed
 }
+
+test_global_use_old_threading_behaviour() {
```

Comment:
This test probably belongs in `test-stbt-py.sh`.

Reply:
We should also test `stbt.frames()`.

Reply:
> This test probably belongs in test-stbt-py.sh.

fixed.